tests/test_suites/SingleMachineTests/test_helgrind_basic.sh

```diff
@@ -49,6 +49,7 @@ cd "${info[mount0]}"
 
 generateFiles
 
-drop_caches
-
-validateFiles
+# FIX(Dave, Guillex): This validateFiles is causing timeouts on some machines. Sometimes it
```

Comment:
In my opinion, this change falls outside the original scope of the PR, especially if the test was already failing prior to these changes.

Another approach might be to create a separate PR to move the original test into one of the `FailingTests` suites. This would allow the reason for disabling the test to be properly documented, making it easier to track in the future.

This way, if another developer decides to investigate and fix the test later, they will have a clear understanding of why it was disabled.